tests/suites/test_suite_pem.data

```diff
@@ -22,26 +22,22 @@ mbedtls_pem_write_buffer_lengths
 PEM read (unencrypted, valid)
 mbedtls_pem_read_buffer:"^":"$":"^\nTWJlZCBUTFM=\n$":"":0:"4d62656420544c53"
 
-PEM read (DES-EDE3-CBC + invalid iv)
```

Comment:
Please keep the DES test cases.

And I think we should backport. We generally backport test coverage improvements if they're relevant and not too hard.

Reply:
We usually don't create the backport until the main PR has two approvals.

Reply:
Please squash the commits before review. We generally keep commits as they come, but [remove x] followed by [restore x] is an unusual case where it's best to squash.

The review includes the commit history, so we'd have to re-review after squashing anyway. As much as possible, we avoid rewriting the history once review has started.

Reply:
Ah, ok, I'd understood “there are no more suggested changes” as after review. Please go on then.